Scripts/Python/plasma/Plasma.py

```diff
@@ -1468,10 +1448,6 @@ def useEAXAcceleration(self,state):
         """Enables or disables EAX sound acceleration (requires hardware acceleration)."""
         pass
 
-    def useHardwareAcceleration(self,state):
-        """Enables or disables audio hardware acceleration."""
-        pass
```

Comment:
Some of these changes don't look related to Python3-ification -- possibly from your audio PR?

Reply:
Yes, I wrote this on top of that

Reply:
Hmm, that does make it harder to review then...  I may hold off until that one is merged.